src/js/multiDomainFirstParties.js

```diff
@@ -253,10 +253,12 @@ var multiDomainFirstPartiesArray = [
   ["mi.com", "xiaomi.com"],
   [
     "microsoft.com",
+
```

Comment:
is this space on purpose?

Reply:
It is, it's the most-recognizable name, the only one that's not alphabetized. Just trying to organize things for readability, I dunno.

Reply:
Oh that makes sense.